### DISCUSSION TITLE
Enter Pem pass phrase

### DISCUSSION THREAD

**Discussion by InfiniteLife** [cesanta/mongoose#2053] [Ask questions, get answers] [answered]
I'm trying to enable https for web server as in tutorial:
```
static void fn(struct mg_connection *c, int ev, void *ev_data, void *fn_data) {
  if (ev == MG_EV_ACCEPT) {
    struct mg_tls_opts opts = {
      .cert = "cert.pem",    // Certificate file
      .certkey = "key.pem",  // Private key file
    };
    mg_tls_init(c, &opts);
  }
```

But when I run the server and try to access it from browser, console running server application promts "Enter PEM pass phrase" and waits for promt. How do I pass file that contains pass phrase to web server, so it wont ask for pem phrase?

**Comment by scaprile** [comment, marked as answer]
#2052
